---
layout: page
title: Thompson Lab Research Interests
---

These are the main areas of research in the Thompson Lab at Michigan State University:
<ul>
<li><a href="#QuantGen">Quantitative genetics</a></li>
<li><a href="#Phenotyping">High throughput phenotyping</a></li>
<li><a href="#Orphan">Miscellaneous Other Cool Science</a></li>
</ul>

<a id="QuantGen" />

## Quantitative Genetics


At its most basic, this area ensures that graduates of the Schnable lab are familiar with modern algorithms to perform basic quantitative genetic tasks including GWAS and genomic selection. However, more advanced work in this area involves developing and applying new statistical approaches that either leverage either the unique features of new phenotypic datasets (for example time-series data from whole mapping populations collected using HTP technologies) or sharing data across related species. 

![CompGen](/images/Science_Images/QGMerged.jpg)

### Recent Lab Publications on Quantitative Genetics

* **Liang Z**, Qiu Y, **Schnable JC** (2018) "[Distinct characteristics of genes associated with phenome-wide variation in maize (*Zea mays*)](https://doi.org/10.1101/534503)" *bioRxiv* doi: 10.1101/534503
* **Miao C**, **Yang J**, **Schnable JC** (2018) "[Optimizing the identification of causal variants across varying genetic architectures in crops.](https://doi.org/10.1111/pbi.13023)" *Plant Biotechnology Journal* doi: 10.1111/pbi.13023 <i>bioRxiv</i> doi: [10.1101/310391](https://doi.org/10.1101/310391)
* <b>Liang Z</b>, Gupta SK, Yeh CT, <b>Zhang Y</b>, <b>Ngu DW</b>, Kumar R, Patil HT, Mungra KD, Yadav DV, Rathore A, Srivastava RK, Gupkta R, <b>Yang J</b>, Varshney RK, Schnable PS, <b>Schnable JC</b>. (2018) "[Phenotypic data from inbred parents can improve genomic prediction in pearl millet hybrids.](https://doi.org/10.1534/g3.118.200242)" <i>G3: Genes Genomes Genetics</i> doi: 10.1534/g3.118.200242

### Funding Supporting Efforts in Quantitative Genetics

* NSF [RII Track-2 FEC: Functional analysis of nitrogen responsive networks in Sorghum.](https://www.nsf.gov/awardsearch/showAward?AWD_ID=1826781&HistoricalAwards=false)
* FFAR [Crops in silico: Increasing crop production by connecting models from the microscale to the macroscale.](http://www.ncsa.illinois.edu/news/story/crops_in_silico_project_awarded_5_million)


<a id="Phenotying"></a>

## High Throughoupt  Phenotyping


This area focuses on developing and deploying new algorithms, tools, and datasets for high throughput plant phenotyping. On the development side we collaborate closely with engineers and statisticians both here at UNL and at other univerisities around the world.

![CompGen](/images/Science_Images/HTPMerged.jpg)

### Recent Lab Publications on High Throughput Phenotyping

* Ge Y, Atefi A, Zhang H, **Miao C**, Ramamurthy RK, **Sigmon B**, Yang J, **Schnable JC** (2019) "High-throughput analysis of leaf physiological and chemical traits with VIS-NIR-SWIR spectroscopy: A case study with a maize diversity panel." *Plant Methods* *(Accepted)*
* Atefi A, Ge Y,  Pitla S, **Schnable JC** (2019) [*In vivo* human-like robotic phenotyping of leaf traits in maize and sorghum.](https://doi.org/10.1016/j.compag.2019.104854) *Computers and Electronics in Agriculture* doi: 10.1016/j.compag.2019.104854

### Funding Supporting Efforts in High Throughput Phenotyping

* NSF [BTT EAGER: A wearable plant sensor for real-time monitoring of sap flow and stem diameter to accelerate breeding for water use efficiency.](https://www.nsf.gov/awardsearch/showAward?AWD_ID=1844563) 
* ARPA-E: In-plant and in-soil microsensors enabled high-throughput phenotyping of root nitrogen uptake and nitrogen use efficiency."
* ARPA-E: Low cost wireless chemical sensor networks."
