<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
    <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
    <div data-badge-type="2" data-doi="10.1101/2020.08.25.266635" data-hide-no-mentions="true" data-hide-less-than="2" class="altmetric-embed" style="float:right;"></div>
* **Meng X**, **Liang Z**, **Dai X**, **Zhang Y**, Mahboub S, **Ngu DW**, Roston RL, **Schnable JC** "[Predicting transcriptional responses to cold stress across plant species](https://doi.org/10.1101/2020.08.25.266635)" *bioRxiv* doi: 10.1101/2020.08.25.266635
<div data-badge-type="2" data-doi="10.3389/fgene.2019.01294" data-hide-no-mentions="true" data-hide-less-than="2" class="altmetric-embed" style="float:right;"></div>
* Jarquin D, Howard R, **Liang Z**, Gupta SK, **Schnable JC**, Crossa J (2020) "[Enhancing hybrid prediction in pearl millet using genomic and/or multi-environment phenotypic information of inbreds.](https://doi.org/10.3389/fgene.2019.01294)" *Frontiers in Genetics* doi: 10.3389/fgene.2019.01294
<div data-badge-type="2" data-doi="10.1101/534503" data-hide-no-mentions="true" data-hide-less-than="2" class="altmetric-embed" style="float:right;"></div>
* **Liang Z**, Qiu Y, **Schnable JC** (2020) "[Genome-phenome wide association in maize and arabidopsis identifies a common molecular and evolutionary signature](https://doi.org/10.1016/j.molp.2020.03.003)" *Molecular Plant* doi: 10.1016/j.molp.2020.03.003 *bioRxiv* doi: 10.1101/534503
<div data-badge-type="2" data-doi="10.1101/730473" data-hide-no-mentions="true" data-hide-less-than="2" class="altmetric-embed" style="float:right;"></div>
* **Dai X**, Xu Z, **Liang Z**, Tu X, Zhong S, **Schnable JC**, Li P (2020) "[Non-homology-based prediction of gene functions.](https://doi.org/10.1002/tpg2.20015)" *The Plant Genome* doi: 10.1002/tpg2.20015  *bioRxiv* doi: 10.1101/730473
<div data-badge-type="2" data-doi="10.34133/2020/7481687" data-hide-no-mentions="true" data-hide-less-than="2" class="altmetric-embed" style="float:right;"></div>
* Wang R, Qiu Y, Zhou Y, **Liang Z**, **Schnable JC** (2020) "[A high-throughput phenotyping pipeline for image processing and functional growth curve analysis.](https://doi.org/10.34133/2020/7481687)" *Plant Phenomics* doi: 10.34133/2020/7481687
<div data-badge-type="2" data-doi="10.1016/j.molp.2017.12.010" data-hide-no-mentions="true" data-hide-less-than="2" class="altmetric-embed" style="float:right;"></div>
* **Liang Z**, **Schnable JC**. (2018) "<a href="https://doi.org/10.1016/j.molp.2017.12.010">Functional Divergence Between Subgenomes and Gene Pairs After Whole Genome Duplications.</a>" <i>Molecular Plant</i> doi: 10.1016/j.molp.2017.12.010
<div data-badge-type="2" data-doi="10.1534/g3.118.200242" data-hide-no-mentions="true" data-hide-less-than="2" class="altmetric-embed" style="float:right;"></div>
* <b>Liang Z</b>, Gupta SK, Yeh CT, <b>Zhang Y</b>, <b>Ngu DW</b>, Kumar R, Patil HT, Mungra KD, Yadav DV, Rathore A, Srivastava RK, Gupkta R, <b>Yang J</b>, Varshney RK, Schnable PS, <b>Schnable JC</b>. (2018) "[Phenotypic data from inbred parents can improve genomic prediction in pearl millet hybrids.](https://doi.org/10.1534/g3.118.200242)" *G3: Genes Genomes Genetics* doi: 10.1534/g3.118.200242
<div data-badge-type="2" data-doi="[10.1101/120428](http://dx.doi.org/10.1101/120428)" data-hide-no-mentions="true" data-hide-less-than="2" class="altmetric-embed" style="float:right;"></div>
*  <b>Lai X</b>,* Behera S,* <b>Liang Z</b>, Lu Y, Deogun JS, <b>Schnable JC</b>. (2017) "[STAG-CNS: An order-aware conserved noncoding sequence discovery tool for arbitrary numbers of species.](http://dx.doi.org/10.1016/j.molp.2017.05.010)" *Molecular Plant* doi: 10.1016/j.molp.2017.05.010 *bioRxiv* preprint doi: [10.1101/120428](http://dx.doi.org/10.1101/120428)
<div data-badge-type="2" data-doi="[10.1101/120303](http://dx.doi.org/10.1101/120303)" data-hide-no-mentions="true" data-hide-less-than="2" class="altmetric-embed" style="float:right;"></div>
*  <b>Zhang Y</b>, <b>Ngu DW</b>, <b>Carvalho D</b>, <b>Liang Z</b>, Qiu Y, Roston RL, <b>Schnable JC</b>. (2017) "[Differentially regulated orthologs in sorghum and the subgenomes of maize](http://dx.doi.org/10.1105/tpc.17.00354)." *The Plant Cell* doi: 10.1105/tpc.17.00354 *bioRxiv* preprint doi: [10.1101/120303](http://dx.doi.org/10.1101/120303)
<div data-badge-type="2" data-doi="[10.1101/169045](https://doi.org/10.1101/169045)" data-hide-no-mentions="true" data-hide-less-than="2" class="altmetric-embed" style="float:right;"></div>
*  <b>Liang Z</b>, Pandey P, Stoerger V, Xu Y, Qiu Y, Ge Y, <b>Schnable JC</b>. (2017) "[Conventional and hyperspectral time-series imaging of maize lines widely used in field trials.](https://academic.oup.com/gigascience/advance-article/doi/10.1093/gigascience/gix117/4656251?guestAccessKey=71c1c32f-78fd-42c0-99a3-38e6c4fd8100)" *GigaScience* doi: 10.1093/gigascience/gix117 *bioRxiv* doi: [10.1101/169045](https://doi.org/10.1101/169045)
<div data-badge-type="2" data-doi="[10.1101/043513](http://dx.doi.org/10.1101/043513)" data-hide-no-mentions="true" data-hide-less-than="2" class="altmetric-embed" style="float:right;"></div>
* <b>Liang Z</b>, <b>Schnable JC</b> (2016) "[RNA-seq based analysis of population structure within the maize inbred B73.](http://dx.doi.org/10.1371/journal.pone.0157942)" *PLoS One* doi: 10.1371/journal.pone.0157942 *bioRxiv* preprint doi: [10.1101/043513](http://dx.doi.org/10.1101/043513)
<div data-badge-type="2" data-doi="" data-hide-no-mentions="true" data-hide-less-than="2" class="altmetric-embed" style="float:right;"></div>
* Chaudhury SD, Steorger V, Samal A, <b>Schnable JC</b>, <b>Liang Z</b>, Yu J. (2016) "<a href="Pubs/Sruti_KDD_FEWS.pdf">Automated vegetative stage phenotyping analysis of maize plants using visible light images.</a>" <i>KDD: Data Science for Food, Energy, and Water</i> San Francisco, CA, USA (Conference Proceedings)
