<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
    <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
    <div data-badge-type="2" data-doi="10.1101/2020.08.25.266635" data-hide-no-mentions="true" data-hide-less-than="2" class="altmetric-embed" style="float:right;"></div>
* **Meng X**, **Liang Z**, **Dai X**, **Zhang Y**, Mahboub S, **Ngu DW**, Roston RL, **Schnable JC** "[Predicting transcriptional responses to cold stress across plant species](https://doi.org/10.1101/2020.08.25.266635)" *bioRxiv* doi: 10.1101/2020.08.25.266635
<div data-badge-type="2" data-doi="10.1186/s12864-020-06824-3" data-hide-no-mentions="true" data-hide-less-than="2" class="altmetric-embed" style="float:right;"></div>
* **Lai X**, Bendix C, **Yan L**, **Zhang Y**, **Schnable JC**, Harmon F (2020) "[Interspecific analysis of diurnal gene regulation in panicoid grasses identifies known and novel regulatory motifs.](https://doi.org/10.1186/s12864-020-06824-3)" *BMC Genomics* doi: 10.1186/s12864-020-06824-3
<div data-badge-type="2" data-doi="10.1101/187575" data-hide-no-mentions="true" data-hide-less-than="2" class="altmetric-embed" style="float:right;"></div>
* **Yan L**, **Raju SKK**, **Lai X**, **Zhang Y**, **Dai X**, Rodriguez O, Mahboub S, Roston RL, **Schnable JC** (2019) "[Parallel natural selection in the cold-adapted crop-wild relative *Tripsacum dactyloides* and artificial selection in temperate adapted maize.](https://doi.org/10.1111/tpj.14376)" *The Plant Journal* doi: 10.1111/tpj.14376 *bioRxiv* doi: 10.1101/187575
<div data-badge-type="2" data-doi="10.1534/g3.118.200242" data-hide-no-mentions="true" data-hide-less-than="2" class="altmetric-embed" style="float:right;"></div>
* <b>Liang Z</b>, Gupta SK, Yeh CT, <b>Zhang Y</b>, <b>Ngu DW</b>, Kumar R, Patil HT, Mungra KD, Yadav DV, Rathore A, Srivastava RK, Gupkta R, <b>Yang J</b>, Varshney RK, Schnable PS, <b>Schnable JC</b>. (2018) "[Phenotypic data from inbred parents can improve genomic prediction in pearl millet hybrids.](https://doi.org/10.1534/g3.118.200242)" *G3: Genes Genomes Genetics* doi: 10.1534/g3.118.200242
<div data-badge-type="2" data-doi="[10.1101/120303](http://dx.doi.org/10.1101/120303)" data-hide-no-mentions="true" data-hide-less-than="2" class="altmetric-embed" style="float:right;"></div>
*  <b>Zhang Y</b>, <b>Ngu DW</b>, <b>Carvalho D</b>, <b>Liang Z</b>, Qiu Y, Roston RL, <b>Schnable JC</b>. (2017) "[Differentially regulated orthologs in sorghum and the subgenomes of maize](http://dx.doi.org/10.1105/tpc.17.00354)." *The Plant Cell* doi: 10.1105/tpc.17.00354 *bioRxiv* preprint doi: [10.1101/120303](http://dx.doi.org/10.1101/120303)
<div data-badge-type="2" data-doi="10.1038/s41598-017-13236-z" data-hide-no-mentions="true" data-hide-less-than="2" class="altmetric-embed" style="float:right;"></div>
*  Washburn JD, <b>Schnable JC</b>, Brutnell TP, Shao Y, <b>Zhang Y</b>, Ludwig M, Davidse G, Pires JC. (2017) "<a href="https://doi.org/10.1038/s41598-017-13236-z">Genome-Guided Phylo-Transcriptomics.</a>" <i>Scientific Reports</i> doi: 10.1038/s41598-017-13236-z
